setup.py

```diff
@@ -15,7 +15,7 @@
         'Bug Tracker': 'https://github.com/Whist-Team/Whist-Core/issues',
     },
     classifiers=[
-        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.9',
```

Comment:
```suggestion
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.9',
```

Reply:
Look here: https://github.com/django/django/blob/main/setup.cfg
And here: https://img.shields.io/pypi/pyversions/Django